src/main/java/be/uantwerpen/minelabs/block/entity/ChargedPointBlockEntity.java

```diff
@@ -0,0 +1,108 @@
+package be.uantwerpen.minelabs.block.entity;
+
+import be.uantwerpen.minelabs.gui.charged_point_gui.ChargedPointBlockScreenHandler;
+import be.uantwerpen.minelabs.inventory.ImplementedInventory;
+import be.uantwerpen.minelabs.util.Tags;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.entity.player.PlayerInventory;
+import net.minecraft.inventory.Inventories;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NbtCompound;
+import net.minecraft.screen.NamedScreenHandlerFactory;
+import net.minecraft.screen.ScreenHandler;
+import net.minecraft.text.Text;
+import net.minecraft.util.collection.DefaultedList;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Direction;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+
+public class ChargedPointBlockEntity extends ChargedBlockEntity implements ImplementedInventory, NamedScreenHandlerFactory {
+
+    private final DefaultedList<ItemStack> INVENTORY = DefaultedList.ofSize(1, ItemStack.EMPTY);
```

Comment:
All caps is used for static fields. Change to lower case.

Reply:
final doesn't indicate const any more than static does. Instance variables are camel case, even when final which just means they can be assigned only once. Things  we want to serve as constants use all caps to indicate this. In this case I really don't think you want the inventory to be a constant.